docs/content/faq.md

```diff
@@ -144,5 +144,34 @@ The Azure mixins are going to be full fledged tools that communicate with the
 Azure APIs, and provide an improved user experience. It all depends on how much
 time you want to invest, and what you are starting from.
 
+## Does the Porter has its own Registry like DockerHub, ACR,Quay etc?
+
+NO Porter does has its own Registry, rather use an OCI compliant artifact store.And now DockerHub is also OCI compliant registry .The list of registries that works with CNAB ie bundles [here](https://porter.sh/compatible-registries/) .
+
+## What is the use of Mixins?
+
+Mixins provide the following features:
+ * Install a tool into the bundle. It handles editing your dockerfile for you.
+ * Mixins can adapt imperative command line tools to work with desired state. For example, the uninstall command should be re-runnable. I should be able to re-run porter uninstall if there was an error and it     should pick up where it left off. Most command line tools are imperative though, and don’t handle errors in a way that works well with that (e.g. returning a failure when you attempt to delete something that   is already deleted).
+ * Mixins provide rich metadata that end-users can use to both understand what a bundle will do, and companies can limit what bundles they allow based on both the mixins present and what functionality of the mi   xin is used.
+ * Mixins help collect outputs from steps, and reuse those as arguments to subsequent steps. For example, the helm mixin can create a database and generate an output with the connection string. The terraform mixin can then use that connection string as an input variable, or even expose that output to the end-user when the bundle finishes running.
+You never have to use mixins, other than the built-in exec. You could write a custom dockerfile and then call a bash script. But most people find that working with mixins is easier.
+
```

Comment:
Mixins provide the following features:
- Install a tool into the bundle. It handles editing your dockerfile for you.
- Mixins can adapt imperative command-line tools to work with the desired state.
   > For example, the uninstall command should be re-runnable. I should be able to re-run the porter uninstall if there was an error and it should pick up where it left off. Most command-line tools are imperative though, and don’t handle errors in a way that works well with that (e.g. returning a failure when you attempt to delete something that is already deleted).

- Mixins provide rich metadata that end-users can use to both understand what a bundle will do, and companies can limit what bundles they allow based on both the mixins present and what functionality of the mixin is used.
- Mixins help collect outputs from steps and reuse those as arguments to subsequent steps. 
  > For example, the helm mixin can create a database and generate an output with the connection string. The terraform mixin can then use that connection string as an input variable, or even expose that output to the end-user when the bundle finishes running.

You never have to use mixins, other than the built-in exec. You could write a custom dockerfile and then call a bash script. But most people find that working with mixins is easier.


***Replace with this!***

Reply:
And use the markdown as I have added above!